watchboi.toml

```diff
@@ -0,0 +1,26 @@
+[http]
+proxy = "127.0.0.1:3080"
+
+[actions.backend]
+base = "backend"
+watch = ["Cargo.toml", "Cargo.lock", "src/"]
+run = ["cargo run"]
+reload = "early"
+
+[actions.frontend]
+base = "frontend"
+watch = [
+    ".babelrc",
+    ".eslintrc.js",
+    "package-lock.json",
+    "package.json",
+    "tsconfig.json",
+    "webpack.config.js",
+    "src/",
+]
+run = [
+    "npx tsc --skipLibCheck",
+    "npx eslint src",
+    "npx webpack --mode=development --display=errors-only",
```

Comment:
Why not use the `npm` scripts here?

Reply:
Because (a) npm scripts always print 4 more lines of noise and (b), the tsc and webpack invocation have special flags only useful for rapid development. `npm run typecheck` shouldn't skip libCheck and `npm run build` is fine to display more information.